web/source/kmwuitoolbar.ts

```diff
@@ -505,7 +506,7 @@ if(!window['keyman']['ui']['name']) {
         var p1=ui.ToolBar_Text['SelectKeyboardPre']+kbd['Name'],p2=ui.ToolBar_Text['SelectKeyboardSuf'];
         if(p1.toLowerCase().indexOf(p2.toLowerCase()) < 0) p1=p1+' '+ p2;
         aNode.title = p1;
-        aNode.onclick = function(event) { return ui.selectLanguage(event, lang) };
+        aNode.onclick = function(event) { return ui.selectLanguage(event, lang) }; // FINDME
```

Comment:
What does `// FINDME` mean? 🤣 
```suggestion
        aNode.onclick = function(event) {
          return ui.selectLanguage(event, lang);
        };
```